test/lib/test/config_test.exs

```diff
@@ -0,0 +1,17 @@
+# Copyright(c) 2015-2020 ACCESS CO., LTD. All rights reserved.
+
+Antikythera.Test.Config.init()
+
+defmodule Antikythera.Test.ConfigTest do
+  use Croma.TestCase
+
+  test "blackbox_test_secret/0 should read BLACKBOX_TEST_SECRET_JSON environment variable" do
+    env_var_name = "BLACKBOX_TEST_SECRET_JSON"
+    original = System.get_env(env_var_name)
+    System.put_env(env_var_name, "{\"foo\": \"bar\"}")
```

Comment:
(optional)
エスケープ文字が不要になります。
```suggestion
    System.put_env(env_var_name, ~S/{"foo": "bar"}/)
```

Reply:
[sigil](https://elixir-lang.org/getting-started/sigils.html) を使うのは良いと思うのですが `/` は正規表現っぽく感じるので、もし使うなら他の記号にしてもらえると嬉しいです

Reply:
No, (at least in Elixir) the delimiter is not used to determine if it is a regular expression or not.
A regular expression is clearly defined to be either `~R` or `~r`.

Reply:
Since [conn_test.exs](https://github.com/access-company/antikythera/blob/master/test/lib/web/conn_test.exs#L163) and [time_test.exs](https://github.com/access-company/antikythera/blob/master/test/lib/util/time_test.exs#L101) use `()` to create a JSON string, I'll use `()` here.

Reply:
Fixed at https://github.com/access-company/antikythera/pull/119/commits/6605e60b5bf1b6cd4efd0a15b049289d485500a2